building/devices/zynqmp.rst

```diff
@@ -0,0 +1,115 @@
+.. _zynqmp_zcu102:
+
+#######################
+QEMU for ZynqMP zcu102
+#######################
+The instructions below show how to run OP-TEE using QEMU for ZynqMP zcu102
+board. 
+
+Setting up the toolchain
+************************
+This build chain heavily relies on Petalinux 2018.2 therefore the first step 
+is to download and install the Petalinux 2018.2 toolchain from the Xilinx
+website.
+By default, the makefile considers that Petalinux is installed in 
+``/opt/Xilinx/petalinux_2018_2``.
+Then, you have to download the zcu102 BSP file from the Xilinx website.
+
+You may have to create a free Xilinx account to proceed with the two
+previous steps.
+
+Configuring and building
+************************
+First, create a new directory which will be used as root directory:
+
+.. code-block:: bash
+
+	$ mkdir -p ~/petalinux-optee
+	$ cd ~/petalinux-optee
+
+Then, copy the zcu102 BSP file into the newly created directory:
+
+.. code-block:: bash
+
+	$ cp ~/Downloads/xilinx-zcu102-v2018.2-final.bsp .
+
+Git clone the ``build`` repository of the OP-TEE project and source
+the Petalinux settings:
+
+.. code-block:: bash
+
+	$ git clone https://github.com/OP-TEE/build
+	$ cd ./build
+	$ source /opt/Xilinx/petalinux_2018_2/settings.sh
+
+Finally, use the following commands to create, patch,
```

Comment:
Line 45-48, please do line breaks on 80 characters (use your preferred method, in vim I'm doing, `set tw=80`, then `SHIFT+v` `jjj ...`, then `g` `q`).